### ISSUE TITLE
Respect "${DESTDIR}"

### ISSUE THREAD

**Issue by vinipsmaker** [majn/telegram-purple#33]
Trying to build a package as follows fails:

```
./configure
make DESTDIR="$pkgdir/"
make DESTDIR="$pkgdir/" install
```

The error is:

```
install -D tg-server.pub /etc/telegram-purple/server.pub
install: cannot create directory /etc/telegram-purple: Permission denied
Makefile:65: recipe for target 'install' failed
make: *** [install] Error 1
```

The script should respect `"${DESTDIR}"` and install the `tg-server.pub` file on `"${DESTDIR}/etc/telegram-purple/server.pub"`.

I'm using version 0.6 and I cannot file mentions to this error on the issue tracker.

**Comment by majn** [comment]
Does this fix work? https://github.com/majn/telegram-purple/commit/87fad25bde5193708726cd43fe5dc29bcc015558

**Comment by vinipsmaker** [comment]
> Does this fix work? 87fad25

Yes, it does work. Thank you.

I'll provide a patched version 0.6 on ArchLinux users repo until the next version is released.
